src/embind/embind.js

```diff
@@ -1947,6 +1947,8 @@ var LibraryEmbind = {
             // Replace the initial unbound-handler-stub function with the appropriate member function, now that all types
             // are resolved. If multiple overloads are registered for this function, the function goes into an overload table.
             if (undefined === proto[methodName].overloadTable) {
+                // Set argCount in case an overload is registered later
+                memberFunction.argCount = argCount - 2;
```

Comment:
I couldn't find any usefulness to className in my tests.
Either it is different, and the method is replaced, or it is the same, and the code creates an overload table only to immediately replace the only method it contains with the newly registered one. Either way, the method previously registered is lost.